### PR TITLE
[ISSUE-3811][Feature] Introduce role management e2e test

### LINKED ISSUE #3811: [Feature] Add Role Management e2e test cases
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Add Role Management e2e test cases:
-  createRole
- deleteRole
- editRole

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhengkezhou1**
## What changes were proposed in this pull request

Issue Number: close #3811 

add basic role management test cases.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (no)

### CODE REVIEW

**Comment by zhengkezhou1** [comment]
@SbloodyS PTAL.

File: streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/cases/RoleManagementTest.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    //TODO: there is no description filed value actual exist
+    //       Awaitility.await()
+    //           .untilAsserted(
+    //               () ->
+    //                   assertThat(roleManagementPage.roleList())
+    //                       .as("Role list should contain newly-created role")
+    //                       .extracting(WebElement::getText)
+    //                       .anyMatch(it -> it.contains(newEditDescription)));
+    }
```
Comment (SbloodyS) [inline streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/cases/RoleManagementTest.java]: I tested it and it was a bug. The description did not showing successfully. Would you like to fix it?

Comment (zhengkezhou1) [reply streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/cases/RoleManagementTest.java]: > I tested it and it was a bug. The description did not showing successfully. Would you like to fix it?

i will fix it.

Comment (SbloodyS) [reply streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/cases/RoleManagementTest.java]: You can open another to fix this bug and uncomment this test in that PR.

**Review by SbloodyS — APPROVED** [review]
LGTM

**Comment by wolfboys** [comment]
Could you please resolve the conflict first? I'd be happy to approve the PR

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**